src/qtpromise/qpromiseglobal.h

```diff
@@ -113,4 +113,14 @@ struct ArgsOf<R (T::*)(Args...) const volatile> : public ArgsTraits<Args...>
 
 } // namespace QtPromisePrivate
 
+
+// qtpromise still supports MSVC 2013 on CI/CD
+// -> MSVC 2013 does not support N2761, hence we need to wrap the noreturn attribute
+#if defined(_MSVC) &&  (_MSVC <= 1800)
+#define NO_RETURN
```

Comment:
This `NO_RETURN` define should be removed since you use `Q_NORETURN` instead.

Reply:
Agreed, I'll change it. What is the projects policy here? New commit OR rebase away in the existing commit?

Reply:
New commit to keep track of review changes (the PR will be squashed on merge).